### DISCUSSION TITLE
Showcasing Photon at DigitalOcean Deploy

### DISCUSSION THREAD

**Discussion by diabhey** [silvia-odwyer/photon#182] [General]
Hello @silvia-odwyer and team,

I’m Abhi, leading the DevRel team at DigitalOcean. On July 9th, I will present at [Deploy](https://www.digitalocean.com/deploy) with a talk titled **Pixels to Cloud: Building a Photo Editor powered by gRPC and DigitalOcean Spaces**. I’ve leveraged your fantastic library to demonstrate photo editing capabilities and mentioned it in my presentation. I will also ensure that the library is linked when our repository goes public.

Thank you once again for this incredible library. If you have the time, I would love for you to join the talk.

Best regards,
Abhi

**Comment by silvia-odwyer** [comment]
@diabhey Hi Abhi,
Wow! I'm delighted that you'll be presenting with a talk that leverages the library. 🎉 I have registered for DigitalOcean Deploy, and will be excited to watch the talk. Wishing you the very best, and thanks again!

**Comment by diabhey** [comment]
@silvia-odwyer : [Here](https://github.com/digitalocean/pixels-to-cloud) is the repository. Feel free to check it out and share it with your community! I look forward to more collaborations in the future :)

**Reply by silvia-odwyer** [reply]
@diabhey I really enjoyed the talk, it was super interesting! I loved the demo at the end, it was very cool to see the images being synced in the various datacentres across the world. I'm glad to have learned more about the storage provided by DigitalOcean Spaces too
I will be sure to link to the repository in the Readme of this library too, so that others can take a look at the repository. Thanks again! :-)
